app.js

```diff
@@ -7,7 +8,7 @@ function School(name, start, finish, degree) {
   this.result = degree;
 
   this.addSchool = function(){
```

Comment:
add your methods to the prototype instead of creating them inside the constructor